### PR TITLE
GPII-3412 - Improve the GCP/README.md

### PR DESCRIPTION
**Pull request by amatas**
This PR adds some notes that can help to the developers in the use of the new infrastructure built on top of GCP.

[Rendered README.md](https://github.com/amatas/gpii-infra/blob/GPII-3412/gcp/README.md)

### CODE REVIEW

File: gcp/README.md [hunk 1]
```diff
@@ -42,8 +51,14 @@ An environment needs some resources created in the organization before the follo
    * The `gpii-infra` clone and the `exekube` clone should be siblings in the same directory (there are some references to `../exekube`).
 1. By default you'll use the RtF Organization and Billing Account.
    * You can use a different Organization or Billing Account, e.g. from a GCP Free Trial Account, with `export ORGANIZATION_ID=111111111111` and/or `export BILLING_ID=222222-222222-222222`.
+1. In the case of using a **dev** environment, be sure that the environment variable `$USER` is set to the same name used to name your dev project at GCP. In case of doubt ask to the ops team.
 1. `cd gpii-infra/gcp/live/dev`
 1. `rake`
+1. If it's the first time that you deploy the infrastructure you will be prompted to verify your identity at Google and allow permissions to your applications to perform modifications in your personal project at GCP. Go to the url shown and copy and paste the token once the application is authorized.
+1. Once finished all the GPII endpoints should be available at https://<service>.<your cluster name>.dev.gcp.gpii.net/
```
Comment (mrtyler) [inline gcp/README.md]: I would add an example here to illustrate: "should be available at https://XXX, e.g. https://preferences.amatas.dev.gcp.gpii.net/".

File: gcp/README.md [hunk 2]
```diff
@@ -42,8 +51,14 @@ An environment needs some resources created in the organization before the follo
...
+1. The dashboard is available through the [Google Cloud Console](https://console.cloud.google.com)
```
Comment (mrtyler) [inline gcp/README.md]: There's a *lot* of stuff on the Console landing page. I would try to provide a more specific link.

File: gcp/README.md [hunk 3]
```diff
@@ -42,8 +51,14 @@ An environment needs some resources created in the organization before the follo
...
+
+##
```
Comment (mrtyler) [inline gcp/README.md]: Delete unused heading.

**Comment by mrtyler** [comment]
LGTM

**Comment by natarajaya** [comment]
LGTM

**Pull request merged**